### ISSUE TITLE
Importing realms with existing organizations

### ISSUE THREAD

**Issue by youssefbennour** [keycloak/keycloak#30824]
### Before reporting an issue

- [X] I have read and understood the above terms for submitting issues, and I understand that my issue may be closed without action if I do not follow them.

### Area

organizations

### Describe the bug

importing realms with existing organizations does not work if organizations are not linked to at least one identity provider.
Logs:
```
ERROR [org.keycloak.services.error.KeycloakErrorHandler] (executor-thread-5) Uncaught server error: java.lang.NullPointerException: Cannot invoke "java.util.List.iterator()" because the return value of "org.keycloak.representations.idm.OrganizationRepresentation.getIdentityProviders()" is null
2024-06-26 14:52:56     at org.keycloak.storage.datastore.DefaultExportImportManager.importOrganizations(DefaultExportImportManager.java:1595)
2024-06-26 14:52:56     at org.keycloak.storage.datastore.DefaultExportImportManager.importRealm(DefaultExportImportManager.java:474)
2024-06-26 14:52:56     at org.keycloak.models.utils.RepresentationToModel.importRealm(RepresentationToModel.java:140)
2024-06-26 14:52:56     at org.keycloak.services.managers.RealmManager.importRealm(RealmManager.java:582)
2024-06-26 14:52:56     at org.keycloak.services.managers.RealmManager.importRealm(RealmManager.java:511)
2024-06-26 14:52:56     at org.keycloak.services.managers.RealmManagerProviderFactory.lambda$postInit$0(RealmManagerProviderFactory.java:54)
2024-06-26 14:52:56     at org.keycloak.services.DefaultKeycloakSessionFactory.publish(DefaultKeycloakSessionFactory.java:91)
2024-06-26 14:52:56     at org.keycloak.storage.ImportRealmFromRepresentationEvent.fire(ImportRealmFromRepresentationEvent.java:50)
2024-06-26 14:52:56     at org.keycloak.storage.datastore.DefaultExportImportManager.importRealm(DefaultExportImportManager.java:171)
2024-06-26 14:52:56     at org.keycloak.services.resources.admin.RealmsAdminResource.importRealm(RealmsAdminResource.java:147)
2024-06-26 14:52:56     at org.keycloak.services.resources.admin.RealmsAdminResource$quarkusrestinvoker$importRealm_aa9defcbd5e1fb410aecd3ab57b280946687671e.invoke(Unknown Source)
2024-06-26 14:52:56     at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
2024-06-26 14:52:56     at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
2024-06-26 14:52:56     at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
2024-06-26 14:52:56     at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:582)
2024-06-26 14:52:56     at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2513)
2024-06-26 14:52:56     at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1538)
2024-06-26 14:52:56     at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:29)
2024-06-26 14:52:56     at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:29)
2024-06-26 14:52:56     at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
2024-06-26 14:52:56     at java.base/java.lang.Thread.run(Thread.java:1583)
```

### Version

25.0

### Regression

- [X] The issue is a regression

### Expected behavior

Successful import of realms along with the defined organizations from .json import file.

### Actual behavior

In case of a startup import, the service crashes and shuts down after the exception is thrown.
In case of manual import, a pop up with a generic message is displayed.

### How to Reproduce?

1 - Create an organization without linking it to an identity provider.
2 - Export the .json realm configuration
3 - Create a new docker container and import the configuration either manually or at startup (using --import-startup command when running the container)

### Anything else?

I was able to proceed with the import by removing the json object from path : 'attributes/organizations'.

**Comment by pedroigor** [comment]
This has been fixed by https://github.com/keycloak/keycloak/pull/30527. Can you check a nightly build?

To workaround the issue you can set an `[]` empty JSON array for identity providers and members.

**Comment by youssefbennour** [comment]
> This has been fixed by #30527. Can you check a nightly build?
> 
> To workaround the issue you can set an `[]` empty JSON array for identity providers and members.

Thanks, I'll also try the nightly build.
